qiskit_nature/second_q/formats/molecule_info.py

```diff
@@ -0,0 +1,59 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""A dataclass storing Molecule information."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Sequence
+
+import numpy as np
+
+from qiskit_nature.units import DistanceUnit
+
+
+@dataclass
+class MoleculeInfo:
+    """A dataclass storing molecule information."""
+
+    symbols: Sequence[str]
+    """The ordered sequence of atoms which make up this molecule."""
+
+    coords: np.ndarray
+    """The XYZ coordinates of the atoms represented as a matrix."""
+
+    multiplicity: int = 1
+    """The multiplicity of the molecule (`= 2 * spin + 1`)."""
+
+    charge: int = 0
+    """The total charge of the molecule."""
+
+    units: DistanceUnit = DistanceUnit.BOHR
+    """The distance unit in which the XYZ coordinates are stored."""
```

Comment:
I default this to Bohr since that's what the `QCSchema` dictates. It's also technically the atomic unit which is what we use throughout the code, too.

I'm open to changing it back, though. Or we make the drivers default to Bohr, too...